config/crd/bases/k8s.raptor.ml_features.yaml

```diff
@@ -1,10 +1,9 @@
----
 apiVersion: apiextensions.k8s.io/v1
```

Comment:
This file is autogenerated, applying the auto-formatter here will force you to reapply everytime you run make manifests